### ISSUE TITLE
Cannot log in using default superadmin account (pass: superadmin)

### ISSUE THREAD

**Issue by richh93** [umeshkumark/mean_bootstrap#1]
mongo says:

GET /login 200 1ms - 12.46kb
Carousel_Routes#getCarouselList
Carousel_DAO#getCarouselList
Carousel_DAO#getCarouselList.Error - MongoError: auth failed
Home_Routes#getCarouselList.Error - MongoError: auth failed
GET /rest/api/home/carousels 200 26ms - 2b
Admin_Routes#authenticateUser
Utility#generateHash.Plain Text - superadmin
Utility#generateHash.Hash - 17c4520f6cfd1ab53d8745e84681eb49
Admin_DAO#authenticateUser Login ID - superadmin
Admin_DAO#authenticateUser.Error - MongoError: auth failed
Admin_Routes#authenticateUser.Error - MongoError: auth failed
POST /rest/api/admin/login/superadmin/superadmin 200 10ms - 2b

When I try to log in, nothing happens.

Thanks

**Comment by umeshkumark** [comment]
Hi Richard, I will look into this issue & get back to you

**Comment by umeshkumark** [comment]
Hi Richard,

Could you check if the Mongo DB credentials you entered in config.json are correct?

I mean are you able to connect to the MongoDB using the credentials in config.json outside the MEAN bootstrap application using command line or any tools like robomongo?

**Comment by richh93** [comment]
Yes when I run the mongod cmd everything works it says listening on port 27...

Thanks
